api/test/integration/env/base/manager/entrypoint.sh

```diff
@@ -8,6 +8,15 @@ for conf_file in /tmp/configuration_files/*.conf; do
   python3 /tools/xml_parser.py /var/ossec/etc/ossec.conf $conf_file
 done
 
+if [ $4 == "standalone" ]; then
+  sed -i -e "/<cluster>/,/<\/cluster>/ s|<disabled>[a-z]\+</disabled>|<disabled>yes</disabled>|g" /var/ossec/etc/ossec.conf
+else
+  sed -i "s:<key>key</key>:<key>9d273b53510fef702b54a92e9cffc82e</key>:g" /var/ossec/etc/ossec.conf
```

Comment:
This is also being done after the if, is that correct? seems duplicate and only working because cluster would be disabled in any case and it won't affect, but does not seem correct.

Reply:
Yes, this was removed but after a wrong solved conflict when rebasing, the code appeared again, solved!